Automattic/wp-calypso
# Social Previews | Update Twitter preview

Completes 1203820933396971-as-1204281808721016/f

## Proposed Changes

This PR updates the Twitter preview component to the latest design from Twitter

## Testing Instructions



* Checkout this PR
* In `wp-calypso/packages/social-previews/` directory, `run pnpm link --global` to expose the package to global `node_modules` for `pnpm`
* In `jetpack/projects/js-packages/publicize-components/ directory`, run `pnpm link --global @automattic/social-previews`
* In `wp-calypso` root, run `yarn` and then `yarn workspace @automattic/social-previews run build`
* Checkout `feature/social-previews-v2` branch on Jetpack repo
* In Jetpack, run `jetpack build plugins/jetpack --no-pnpm-install`
* Start writing a new post with JT site via Jetpack
* Click Jetpack icon on the sidebar
* Under "Social Previews", click on "Preview"
* Confirm that the preview of Twitter is as per the design

<table>
	<thead>
		<tr>
			<th>BEFORE</th>
			<th>AFTER</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<th colspan="2">Single tweet</th>
		</tr>
		<tr>
			<td><img width="1411" alt="Screenshot 2023-04-07 at 6 23 05 PM" src="https://user-images.githubusercontent.com/18226415/230612394-fb033162-db73-4493-91d3-57390fa330b2.png"></td>
			<td><img width="1412" alt="Screenshot 2023-04-07 at 5 39 07 PM" src="https://user-images.githubusercontent.com/18226415/230611562-63c48dc8-201a-440a-abb0-944e1a88d1dd.png"></td>
		</tr>
		<tr>
			<th colspan="2">Thread</th>
		</tr>
		<tr>
			<td><img width="1409" alt="Screenshot 2023-04-07 at 6 22 34 PM" src="https://user-images.githubusercontent.com/18226415/230612438-aea52f3c-740f-47a7-90e4-a7aa40e9ac6d.png"></td>
			<td><img width="1411" alt="Screenshot 2023-04-07 at 5 40 44 PM" src="https://user-images.githubusercontent.com/18226415/230611536-c75f33b6-bcfe-4444-9e99-72923d14d4c2.png"></td>
		</tr>
	</tbody>
</table>

## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204281808721016
